packages/wallet/wdk/src/sequence/manager.ts

```diff
@@ -403,7 +403,9 @@ export class Manager {
       cron: new Cron(shared),
       logger: new Logger(shared),
       devices: new Devices(shared),
-      guard: new Guard(shared),
+      guard: shared.sequence.guardUrl
+        ? new Guard.Sequence.GuardSigner(shared.sequence.guardUrl, shared.sequence.guardAddress)
+        : new Guard.Local.GuardSigner(Secp256k1.randomPrivateKey()),
```

Comment:
Can we support a fixed local key? Maybe as another env var. Otherwise the guard will be different every time you restart the app making local dev harder.